resources/views/admin/accounts.blade.php

```diff
@@ -5,83 +5,90 @@
-                    <tr class="hover:bg-gray-100 dark:hover:bg-gray-900">
-                        <td class="px-4 py-2 whitespace-nowrap text-sm font-medium text-gray-800 dark:text-white">
-                            {{$account['email']}}
-                        </td>
-                        <td class="px-4 py-2 whitespace-nowrap text-sm font-medium text-gray-800">
-                            <div class="relative inline-block text-left">
-
-                                @php
-                                    $oldRole = $account->roles->first() ? $account->roles->first()->name : null;
-                                    $translatedOldRole = $oldRole ? __('roles.' . $oldRole) : null;
-                                @endphp
-
-                                <label hidden>{{ __('accounts.role')  }}</label>
-                                <select data-account-email="{{ $account->email }}"
-                                        data-old-role="{{ $account->roles->first()->name }}"
-                                        class="block appearance-none w-full border border-gray-300 py-2 px-4 pr-8 rounded leading tight focus:outline-none focus:border-blue-500 dark:bg-gray-800 dark:text-gray-400 dark:border-gray-700">
-
-                                    @foreach (trans('roles') as $role => $translatedRole)
-                                        <option
-                                                value="{{ $role }}" {{ $translatedOldRole === $translatedRole ? "selected" : "" }}>
-                                            {{ __('roles.' . $role) }}
-                                        </option>
-                                    @endforeach
-
-                                </select>
-                            </div>
-                        </td>
-                    </tr>
-                @endforeach
-
-                </tbody>
-            </table>
+    <div class="flex flex-col">
+            <div class="p-1.5 min-w-full inline-block align-middle">
+                    <table class="min-w-full divide-y divide-gray-200 dark:divide-gray-700">
+                        <thead>
+                        <tr>
+                            <th scope="col"
+                                class="w-4/5 px-6 py-3 text-start text-xs font-medium text-gray-500 uppercase">
+                                @sortablelink('email', __('accounts.email'))
+                            </th>
+                            <th scope="col"
+                                class="w-1/5 px-6 py-3 text-start text-xs font-medium text-gray-500 uppercase">
+                                {{ __('accounts.role') }}
+                            </th>
+                        </tr>
+                        </thead>
+                        <tbody class="divide-y divide-gray-200 dark:divide-gray-700">
+                        @foreach($accounts as $account)
+                            @if($account->email !== Auth::user()->email)
```

Comment:
Ik denk dat er bij de merge iets mis is gegaan, maar deze logica kan dus beter in de controller

Reply:
oh, dan is er indd iets mis gegaan, want dat had ik wel weggehaald. Ik zal het even aanpassen